Expecto.Tests/PerformanceTests.fs

```diff
@@ -25,13 +26,17 @@ let performance =
     ) |> assertTestFailsWithMsgContaining "same"
 
     testCase "sha256 versus md5" (fun _ ->
+      if RuntimeInformation.IsOSPlatform(OSPlatform.Linux) then
+        skiptest "Doesn't hold true on this platform for unclear reasons"
       Expect.isFasterThan
         (runSHA256 >> ignore |> repeat10)
         (runMD5 >> ignore |> repeat10)
         "SHA256 is faster than MD5 should fail"
     ) |> assertTestFailsWithMsgContaining "slower"
 
     testCase "md5 versus sha256" <| fun _ ->
+      if RuntimeInformation.IsOSPlatform(OSPlatform.Linux) then
+        skiptest "Doesn't hold true on this platform for unclear reasons"
```

Comment:
I was hesitant to remove them, since I don't quite understand the purpose of the tests.
But it also seems like the same functionality is already covered by [these tests](https://github.com/haf/expecto/blob/d9c19fbd1d0868ed81b4ab27143f65bd3bec00cb/Expecto.Tests/Tests.fs#L1453).

I think I'll just go ahead and remove them

Reply:
Done